style.css

```diff
@@ -0,0 +1,732 @@
+    backdrop-filter: blur(10px);
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100vh;
+    transform: translateX(100%);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    transition: all 0.5s ease-in;
+    opacity: 0;
+    pointer-events: none;
+    visibility: hidden;
+    z-index: 1000;
+  }
+
+  .nav-open .main-nav {
+    opacity: 1;
+    pointer-events: auto;
+    visibility: visible;
+    transform: translateX(0);
+  }
+
+  /* MOBILE */
+  .icon-mobile-nav[name="close"] {
+    display: none;
+  }
+
+  .nav-open .icon-mobile-nav[name="close"] {
+    display: block;
+  }
+
+  .nav-open .icon-mobile-nav[name="menu"] {
+    display: none;
+  }
+
+  .main-nav-list {
+    display: flex;
+    flex-direction: column;
+    gap: 3.6rem;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+  }
+
+  .icon-mobile-nav {
+    height: 4.8rem;
+    width: 4.8rem;
+    color: #333;
+  }
+
+  .header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: #fff;
+    height: 4.8rem;
+    padding: 0 2.4rem;
+    position: relative;
+  }
+
+  .program-link {
+    display: none;
+  }
+}
+
+.margin-sect {
+  margin-top: 9.6rem;
+}
+
+/* Navigation */
+
+.primary-header {
+  font-weight: 400;
+  font-size: 1.8rem;
+  line-height: 3.8rem;
+  color: #ec5242;
+}
+
+.headline-info {
+  text-transform: uppercase;
+  font-size: 2.8rem;
+  color: #ec5242;
+  font-weight: 700;
+  margin-bottom: 1.2rem;
+}
+
+.concert-info {
+  font-size: 1.2rem;
+  font-weight: 400;
+  line-height: 1.6;
+  letter-spacing: 0.75px;
+  border: 2px solid #fff;
+  padding: 1.2rem;
+  margin-bottom: 2.4rem;
+}
+
+.music-icon {
+  height: 3.2rem;
+  width: 3.2rem;
+  color: #ec5242;
+}
+
+.main-nav span {
+  display: block;
+  font-size: 1.6rem;
+}
+
+.section-headline {
+  background-image: url(assets/img/hero-bg.png);
+  background-size: cover;
+  padding-top: 7rem;
+  padding-bottom: 5rem;
+}
+
+.concert-duration {
+  font-weight: 700;
+  font-size: 3.2rem;
+  color: #666;
+  margin-bottom: 1.2rem;
+  line-height: 1.2;
+}
+
+.concert-location {
+  font-weight: 400;
+  font-size: 1.4rem;
+}
+
+/* Program-Section */
+
+.section-program {
+  background-image: url(assets/img/checked-pattern.png);
+  background-repeat: repeat;
+  background-color: rgb(39, 42, 50);
+  padding: 2rem 0;
+  text-align: center;
+}
+
+.heading-tertiary {
+  color: #fff;
+  text-transform: capitalize;
+  font-size: 2rem;
+  margin: 2rem 2rem 4rem 2rem;
+  position: relative;
+}
+
+.heading-tertiary::before {
+  content: "";
+  display: block;
+  position: absolute;
+  bottom: -35%;
+  width: 5%;
+  left: 47%;
+  border-bottom: 1px solid #ec5242;
+}
+
+.program-details {
+  display: flex;
+  flex-direction: row;
+  gap: 1.8rem;
+  align-items: center;
+  justify-content: center;
+  background-color: #e6e6e625;
+  margin: 1.4rem;
+  padding: 1.2rem;
+}
```

Comment:
- [ ] Kindly make sure to align the card's content correctly in order to meet the provided design. :wink: 

    - ### YOUR DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/203547072-4ab00e67-3d90-4cf4-ab4a-079a8c296463.png)

    - ### PROVIDED FIGMA DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/203546959-3e955a90-d901-42db-8547-122034db8a2d.png)